test/functional/bulk.test.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
+const { withClient } = require('./shared');
 const test = require('./shared').assert,
   setupDatabase = require('./shared').setupDatabase,
   expect = require('chai').expect;
 
 const MongoError = require('../../src/error').MongoError;
 const ignoreNsNotFound = require('./shared').ignoreNsNotFound;
```

Comment:
```suggestion
const { withClient, setupDatabase, ignoreNsNotFound } = require('./shared');
const test = require('./shared').assert;
const { expect } = require('chai');
const { MongoError } = require('../../src/error');
```

Another one of those, "I know its not yours but while you're here if you don't mind" changes 😅